source/atlas-vector-search.txt

```diff
@@ -0,0 +1,133 @@
+      '{"plot":"Hoping to alter the events of the past, a 19th century inventor instead travels 800,000 years into the future, where he finds humankind divided into two warring races.","title":"The Time Machine","score":0.7801066637039185}'
+      `{"plot":"After using his mother's newly built time machine, Dolf gets stuck involuntary in the year 1212. He ends up in a children's crusade where he confronts his new friends with modern techniques...","title":"Crusade in Jeans","score":0.7789170742034912}`
+      '{"plot":"An officer for a security agency that regulates time travel, must fend for his life against a shady politician who has a tie to his past.","title":"Timecop","score":0.7771612405776978}'
+      '{"plot":"A time-travel experiment in which a robot probe is sent from the year 2073 to the year 1973 goes terribly wrong thrusting one of the project scientists, a man named Nicholas Sinclair into a...","title":"A.P.E.X.","score":0.7730885744094849}'
+      `{"plot":"Agent J travels in time to M.I.B.'s early days in 1969 to stop an alien from assassinating his friend Agent K and changing history.","title":"Men in Black 3","score":0.7712380886077881}`
+      '{"plot":"Bound by a shared destiny, a teen bursting with scientific curiosity and a former boy-genius inventor embark on a mission to unearth the secrets of a place somewhere in time and space that exists in their collective memory.","title":"Tomorrowland","score":0.7669923901557922}'
+      '{"plot":"With the help of his uncle, a man travels to the future to try and bring his girlfriend back to life.","title":"Love Story 2050","score":0.7649372816085815}'
+      '{"plot":"A dimension-traveling wizard gets stuck in the 21st century because cell-phone radiation interferes with his magic. With his home world on the brink of war, he seeks help from a jaded ...","title":"The Portal","score":0.7640786170959473}'
+
+This query uses the ``$vectorSearch`` stage to:
+
+- Search for the specified term in the ``plot_embedding`` field
+
+- Set the number of nearest neighbors used in the search to 150 by using the
+  ``numCandidates`` option
+
+- Return a maximum of 10 documents from the query using the ``limit`` option
+
+It uses the ``$project`` stage to:
+
+- Only include the movie ``plot`` and ``title`` fields in the results
+
+- Add a ``score`` field to show the relevance of each result to the search term
+
+Additional Information
+----------------------
+
+To see more Atlas Vector Search tutorials for the {+driver-short+}, see the :atlas:`Atlas
+Vector Search tutorials </atlas-vector-search/tutorials/>` in the Atlas documentation.
+
+To learn more about Atlas Vector Search, see the :atlas:`Atlas Vector Search </atlas-vector-search/>`
+guides in the Atlas documentation.
```

Comment:
```suggestion
To learn more about the syntax of the ``$vectorSearch`` pipeline stage,
see the Syntax and Fields sections of the 
:atlas:`Create and Run Queries </atlas-vector-search/vector-search-stage/#syntax>`
guide in the Atlas Vector Search section of the MongoDB Atlas documentation.
```
I would remove the link to the general AVS docs since there is already one above and add this link for the syntax documentation instead